### ISSUE TITLE
Users would be nice...

### ISSUE THREAD

**Issue by outline4** [iainurquhart/craft_introvert#9]
I have a case where I've changed a section to users. Now it would be nice if you could choose "users" as well.

**Comment by SimonEast** [comment]
Yes, we'll be needing this feature too.  If no one has attempted adding it, our dev team might look into it.
